### PR TITLE
Refactored Awesome books with classes

### PR DESCRIPTION
**Pull request by Asiimwe-Mark**
We refactored our code to use classes and objects. We followed the following requirements.

Project requirements
Use JavaScript classes instead of objects and arrays.
You can use objects and arrays but do it inside classes.
Create class methods to add and remove books.
Maintain all of the original functionality of the application (i.e. no bugs were introduced while refactoring).
Don't use any library or framework, use only plain JavaScript.
Add CSS styles to the application to make it match this wireframe:
You can use the styles of your choice (colors, fonts, backgrounds, etc.), but try to follow the general layout.
Concentrate on the general layout, rather than the details.

### CODE REVIEW

File: src/js/script.js [hunk 1]
```diff
@@ -3,96 +3,78 @@ const titleInput = document.getElementById('title-input');
...
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
+
+class UI {
+  static loadBooks() {
+    const storedBooks = JSON.parse(localStorage.getItem('books'));
```
Comment (joskalenda) [inline src/js/script.js:19]: - [ ] Nice work @team, however, there are 2 items that are present anytime before and after you add or remove a book, please make sure you fix this and will be good to go

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @As1imwe-Mark , @Rebelzob , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Great design
- [ ].. ✅ Clean code
- [ ]... ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Asiimwe-Mark** [comment]
Hello @joskalenda, I have honestly not understood the required change because when I load the page in other browsers, I don't see the items you're saying that appear. Please elaborate more. Thank you

File: src/js/script.js (hunk 1 above)
Comment (Rebelzob) [reply src/js/script.js:19]: Hello @joskalenda here is a video on how it is working for us in our browsers 

https://www.loom.com/share/59872e80ea3b4b35abf09c26da86bbf6

**Review by levy002 — APPROVED** [review]
## Hello @Rebelzob & @As1imwe-Mark 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter error :heavy_check_mark: 
- You app is working well as expected :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**